search-commons/src/test/resources/resource_urls.json

```diff
@@ -184,8 +184,8 @@
   "SCIENTIFIC_REPORT_PERIOD_SINCE=2019": 3,
   "SCIENTIFIC_VALUE=Unassigned": 2,
   "SCIENTIFIC_VALUE=LevelTwo": 1,
-  "SCIENTIFIC_VALUE=LevelOne,LevelTwo": 4,
-  "SCIENTIFIC_VALUE=LevelOne,LevelTwo,Unassigned": 6,
+  "SCIENTIFIC_VALUE=LevelOne,LevelTwo": 6,
+  "SCIENTIFIC_VALUE=LevelOne,LevelTwo,Unassigned": 8,
```

Comment:
These numbers are correct, I have updated them yesterday to make two tests pass.